src/applications/disability-benefits/all-claims/pages/ancillaryFormsWizardIntro.js

```diff
@@ -0,0 +1,18 @@
+import { ancillaryFormsWizardDescription } from '../content/ancillaryFormsWizardIntro';
+
+export const uiSchema = {
+  'ui:title': 'Additional disability benefits',
+  'view:ancillaryFormsWizardIntro': {
+    'ui:description': ancillaryFormsWizardDescription,
+  },
+};
+
+export const schema = {
```

Comment:
Could just put this whole page on the `formConfig`, since that seems to be what we've done for other empty schema pages.

Reply:
Yeah, I noticed that after I wrote this. I'll move it so we don't clutter up the `pages/index.js` file _too_ much more than we need to. :grimacing: